Dockerfile

```diff
@@ -31,13 +31,13 @@
 #
 
 # Using debian 10 as base image.
-FROM debian:10
+FROM opensuse/leap:15.2
```

Comment:
not super sure about the distro choice (just an issue regarding the image size)

Reply:
I think it's better to stick to Debian here. Or Ubuntu, at least.

Reply:
Even better alpine

Reply:
Debian is not an option as the meson version is not working. I'll check alpine to try to limit its size.

Reply:
Might try installing from pip as suggested by @ret2libc

Reply:
Alpine with pip should work, yeah!